src/AuthContext.tsx

```diff
@@ -142,7 +142,7 @@ export const AuthProvider: FC<PropsWithChildren<AuthProviderProps>> = ({
   return (
     <AuthContext.Provider
       value={{
-        signIn: async (args: unknown): Promise<void> => {
+        signIn: async (args: any): Promise<void> => {
```

Comment:
Are we sure we want to use `any` here? If the `args` are not know, they should probably stay `unknown`. It would however be nicer to have some types here, is that something `oidc-client-ts` now provides?